include/pacemaker.h

```diff
@@ -244,6 +244,22 @@ int pcmk_simulate(xmlNodePtr *xml, pcmk_scheduler_t *scheduler,
                   const char *input_file, const char *graph_file,
                   const char *dot_file);
 
+/*!
+ * \brief Verify that a CIB is error-free or output errors and warnings
+ *
+ * This high-level function essentially implements crm_verify(8). It operates
+ * on an input CIB file, which can be inputted through one of several ways. It
+ * can either write out XML-formatted output or plaintext output.
+ *
+ * \param[in,out] xml          The destination for the result, as an XML tree
+ * \param[in]     cib_source   Source of the CIB: 
+ *                             NULL -> use live cib, "-" -> stdin
+ *                             "<..." -> xml str, otherwise -> xml file name
+ *
+ * \return Standard Pacemaker return code
+ */
+int pcmk_verify(xmlNode *xml, const char *cib_source);
```

Comment:
The first argument needs to be `xmlNodePtr *`(aka `xmlNode **`) (not just the contents of the `xmlNode*` but the pointer itself will be reassigned)